### ISSUE TITLE
[Feature request] Subfolder thumbnails

### ISSUE THREAD

**Issue by rikako3** [YACReader/yacreader#111]
On YACReader Library, when you select a folder in your library that is composed of subfolders (which in turn may contain library galleries, or more subfolders), you only get a plain text list of subfolders, even in grid view.

Would it be possible to display a regular thumbnail instead (for example, using the first gallery that is found inside the subfolder)? I think it would make the library easier to navigate.

**Comment by Commander-Kaiser** [comment]
Yes, please. Or maybe have it look for an image file of a certain name and use that as the preview image like in ubooquity.
